docs/developer/decision-records/2022-08-15-code-quality-tooling/README.md

```diff
@@ -0,0 +1,13 @@
+# Code Quality
+
+## Decision
+
+Use [Checkstyle](https://checkstyle.sourceforge.io/) and [CodeQL](https://codeql.github.com/) static code analysis tools to guarantee adherence to coding standards and discover potential vulnerabilities.
+
+## Rationale
+
+Both Checkstyle and CodeQL are already in use in the EDC framework, so these tools are the natural choice in related projects like the Identity Hub as well.
```

Comment:
```suggestion
Both Checkstyle and CodeQL are already in use in the EDC framework, so these tools are the natural choice in related projects like the MinimumViableDataspace as well.
```

Reply:
Applied similar feedback from Marc. Resolving this. Thanks